src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Sdk.CurrentVersion.targets

```diff
@@ -0,0 +1,395 @@
+﻿<Project>
```

Comment:
This is from Design.targets minus the reference to CodeGeneration. I'll point out the changes I made.

Reply:
You could also have this file import the codegeneration targets (if defined). Then the SDK has precise control over the import order.